### ISSUE TITLE
Error installing the debugger is a preview1 version of the .NET CLI is installed

### ISSUE THREAD

**Issue by Ethan-VisualVocal** [OmniSharp/omnisharp-vscode#618]
More of an FYI than a bug report: 

It wasn't obvious to me that I needed to install an update for .NET Core to get this working.  I had previously been running the Legacy C# plugin for Unity dev, but had a dotnet 1.0.0--preview1-###### build. 

**Solution:** Go to https://www.microsoft.com/net/core#macos, follow instructions & install the latest (ended up being a preview2 for me at the time of writing this).  Uninstall Legacy C# plugin, restart VSCode, install this C# plugin, restart VSCode.  Success!

Error log below is what I was seeing, for others like me searching for a solution:
```
Creating /Users/ethan/.vscode/extensions/ms-vscode.csharp-1.3.0/coreclr-debug/project.json
Telemetry is: Enabled

log  : Restoring packages for /Users/ethan/.vscode/extensions/ms-vscode.csharp-1.3.0/coreclr-debug/project.json...

error: System.Security.Cryptography.Algorithms 4.2.0 provides a compile-time reference assembly for System.Security.Cryptography.Algorithms on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.

error: System.Net.Http 4.1.0 provides a compile-time reference assembly for System.Net.Http on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.
error: System.AppContext 4.1.0 provides a compile-time reference assembly for System.AppContext on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.
error: System.Linq 4.1.0 provides a compile-time reference assembly for System.Linq on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.
error: System.Linq.Expressions 4.1.0 provides a compile-time reference assembly for System.Linq.Expressions on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.
error: System.Security.Cryptography.X509Certificates 4.1.0 provides a compile-time reference assembly for System.Security.Cryptography.X509Certificates on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.
error: System.Text.RegularExpressions 4.1.0 provides a compile-time reference assembly for System.Text.RegularExpressions on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.

error: One or more packages are incompatible with .NETCoreApp,Version=v1.0 (osx.10.11-x64).

info : Committing restore...

log  : Writing lock file to disk. Path: /Users/ethan/.vscode/extensions/ms-vscode.csharp-1.3.0/coreclr-debug/project.lock.json

log  : /Users/ethan/.vscode/extensions/ms-vscode.csharp-1.3.0/coreclr-debug/project.json
log  : Restore failed in 1647ms.

Error: 

Error: Errors in /Users/ethan/.vscode/extensions/ms-vscode.csharp-1.3.0/coreclr-debug/project.json

Error:     System.Security.Cryptography.Algorithms 4.2.0 provides a compile-time reference assembly for System.Security.Cryptography.Algorithms on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.

Error:     System.Net.Http 4.1.0 provides a compile-time reference assembly for System.Net.Http on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.

Error:     System.AppContext 4.1.0 provides a compile-time reference assembly for System.AppContext on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.
    System.Linq 4.1.0 provides a compile-time reference assembly for System.Linq on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.
    System.Linq.Expressions 4.1.0 provides a compile-time reference assembly for System.Linq.Expressions on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.
    System.Security.Cryptography.X509Certificates 4.1.0 provides a compile-time reference assembly for System.Security.Cryptography.X509Certificates on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.
    System.Text.RegularExpressions 4.1.0 provides a compile-time reference assembly for System.Text.RegularExpressions on .NETCoreApp,Version=v1.0, but there is no run-time assembly compatible with osx.10.11-x64.
    One or more packages are incompatible with .NETCoreApp,Version=v1.0 (osx.10.11-x64).


NuGet Config files used:

    /Users/ethan/.vscode/extensions/ms-vscode.csharp-1.3.0/coreclr-debug/NuGet.config



Feeds used:
    https://api.nuget.org/v3/index.json

dotnet exited with error code 1
```

**Comment by gregg-miskelly** [comment]
Thanks for the report @Ethan-VisualVocal. 
@chuckries do you think we should block installing if the .NET CLI version we get back in prievew1?

**Comment by chuckries** [comment]
@gregg-miskelly sure. Just a raw `dotnet --version` check after we verify that it's on the path?

**Comment by gregg-miskelly** [comment]
@chuckries since we verify something is on the path by just executing `dotnet` I might suggest that we instead execute `dotnet --version` and check both that it succeeds and that the version number is >= preview2.

**Comment by chuckries** [comment]
@gregg-miskelly good call, I will take this over then.

**Comment by chuckries** [comment]
Fixed with #715. This will be in 1.4 release.
